release-notes/swan-lake-alpha5-release-note.md

```diff
@@ -47,7 +47,191 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://githu
 
 #### New Features
 
+###### Support for configure variables with records of records, record arrays and multi dimensional array types with Config.toml.
+
+**Record of records**
```

Comment:
Shall we say `Record with record field type` or `Record with fields of type record`?

Reply:
This is too long for a heading. Is it possible to summarize and make it shorter?

E.g., "Support for configurable variables with records and arrays"?

Reply:
We already have support for records and arrays. These are some complex cases related to records and arrays. Maybe I can have 3 headings.

1) Support for configurable variables with records with fields of type record
2) Support for configurable variables with arrays with fields of type record
2) Support for configurable variables with multidimensional arrays

Reply:
How about `Improved support for composite configurables`

Reply:
Then we need to consider these as improvements than new features. That should be Ok I think.

Reply:
BTW i have removed the main heading and broken it into 3 short headings.

Reply:
+1